change XML to qmd extension metadata (#6038)
qgis/QGIS-Documentation: docs/user_manual/working_with_vector/vector_properties.rst

tab and helps you identify potential issues related to the form. You can then
either fix them or ignore them.

> Metadata are currently saved in the project file. It can also be saved as a

Reviewer: ```suggestion
Metadata are currently saved in the project file. They can also be saved in a
```

either fix them or ignore them.

> Metadata are currently saved in the project file. It can also be saved as an

Reviewer: ```suggestion
Metadata are currently saved in the project file. It can also be saved as a
```
